<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7704" started-at="2016-12-03T11:27:54Z" finished-at="2016-12-03T11:28:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7704" started-at="2016-12-03T11:27:54Z" finished-at="2016-12-03T11:28:02Z">
      <class name="com.leads.test.web.LoginTest">
        <test-method status="PASS" signature="loadUrl()[pri:0, instance:com.leads.test.web.LoginTest@5fd4f8f5]" name="loadUrl" is-config="true" duration-ms="2373" started-at="2016-12-03T17:27:54Z" finished-at="2016-12-03T17:27:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadUrl -->
        <test-method status="PASS" signature="checkLogin()[pri:0, instance:com.leads.test.web.LoginTest@5fd4f8f5]" name="checkLogin" duration-ms="5293" started-at="2016-12-03T17:27:56Z" finished-at="2016-12-03T17:28:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method status="FAIL" signature="checkTitle()[pri:0, instance:com.leads.test.web.LoginTest@5fd4f8f5]" name="checkTitle" duration-ms="18" started-at="2016-12-03T17:28:02Z" finished-at="2016-12-03T17:28:02Z">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<Select a Flight...> but was:<Welcome...>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<Select a Flight...> but was:<Welcome...>
	at junit.framework.Assert.assertEquals(Assert.java:81)
	at junit.framework.Assert.assertEquals(Assert.java:87)
	at com.leads.test.web.LoginTest.checkTitle(LoginTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTitle -->
        <test-method status="PASS" signature="closeDriver()[pri:0, instance:com.leads.test.web.LoginTest@5fd4f8f5]" name="closeDriver" is-config="true" duration-ms="1" started-at="2016-12-03T17:28:02Z" finished-at="2016-12-03T17:28:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
      </class> <!-- com.leads.test.web.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
